doc/api/http2.md

```diff
@@ -1146,7 +1146,8 @@ of the given file:
 
 If an error occurs while attempting to read the file data, the `Http2Stream`
 will be closed using an `RST_STREAM` frame using the standard `INTERNAL_ERROR`
-code.
+code. The onError callback will be called before send if is defined, otherwise
```

Comment:
Nit: can you please add backticks (`) around `onError`?

Reply:
Of course, I will fix that and I rebase on master too.